### ISSUE TITLE
[Slash] Pagination should have appropriate aria-label

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/design-system#938]
For the pagination, there's no way to set some aria-label to distinguish different paginations.
For example if we have two paginations inside the same page, there is no possibility to set an aria-label on the component to separate them.

Also, the page link inside the pagination have no proper aria-label as well.
If we have page 1, 2 and 3, it will be shown as 1, 2 and 3 links to users giving zero indication of what exactly are those links.

**Comment by charles-gilles** [comment]
Je prends :)

**Comment by Enenche23** [comment]
I'm a Seasoned web Developer and I feel that this task should be a piece of cake
